scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/password/PasswordChangeController.java

```diff
@@ -0,0 +1,19 @@
+package edu.asu.diging.scriptoocloud.web.password;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.simpleusers.web.forms.UserForm;
+ 
+@Controller
+public class PasswordChangeController{
+
+ 	@RequestMapping(value = "/password/change", method=RequestMethod.GET)
+    public String get(Model model) {
+		model.addAttribute("user", new UserForm());
+		return "/password/change";	
+    }   
```

Comment:
formatting

Reply:
Hey Jason, I figured out the implementation.  This Controller is not needed in the application (the Controller from simple-users takes care of Post and Get). See my comment on ../password/change.html as well.